command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 01:19:29PM -0700, Mrunal Patel wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
> 
> I think that we should require a runtime to implement the CLI…

And note that a runtime can ignore this, not be certified, and still
be compliant with *other* parts of the spec.